server/graphql/v1/mutations/orders.js

```diff
@@ -1047,6 +1047,15 @@ export async function addFundsToCollective(order, remoteUser) {
 
   try {
     await libPayments.executeOrder(remoteUser || user, orderCreated);
+
+    // Check if the maximum fund limit has been reached after execution
```

Comment:
Can you extract that in an helper function?

Reply:
This is not what I was thinking about. It was more

```
 await libPayments.executeOrder(remoteUser || user, orderCreated);

await handleHostPlansLimit()
```

```
function handleHostPlansLimit() {
    const hostPlan = await host.getPlan();
    if (hostPlan.addedFunds > hostPlan.addedFundsLimit) {
```

Reply:
@flickz check that comment, the goal is to minimize the amount of specific code in the mutation.